pkg/client/dtclient/internal/respons/schema_detail.go

```diff
@@ -0,0 +1,28 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package respons
```

Comment:
```suggestion
package response
```

Reply:
By purpose it should be unexported, and at the same time exported for unmarshaling! That's the reason why I put them in the `internal` package

Let's see will the future say is it the go-to design. Or is it overdesigned?